packages/pangraph/src/reconsensus/reconsensus.rs

```diff
@@ -87,8 +91,11 @@ fn reconsensus_mutations(block: &mut PangraphBlock) -> Result<(), Report> {
       let subs_at_pos: Vec<_> = edit.subs.iter().filter(|s| s.pos == pos).cloned().collect();
       match subs_at_pos.len() {
         0 => {
-          edit.subs.push(Sub::new(pos, original));
```

Comment:
This fixed the bug: the substitution should be applied only if the position is not deleted.